docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -74,3 +74,61 @@ The summary should be understandable by a non-technical person, and not contain
 ### No weird formatting
 
 The summary should not have any weird formatting or punctuation. It shouldn't end with a full stop.
+
+## Description
+
+### Don't repeat the summary
+
+The description should not just repeat or rephrase the summary. The summary is a kind of slogan or advertisement for the app, while the description should go into more detail about the purpose of the app, which features it has, and what makes it unique.
+
+### Not too short
+
+The description should be substantially longer than the summary. Something like 3-6 lines of text at default line widths (around 70 characters) is a good length for most apps, for very large apps up to about 10 lines.
+
+### No super long lists
+
+Avoid feature lists with more than 10 items, they are difficult to scan and make the app store listing unwieldy. Instead, try to group points or use paragraphs instead of bullet points.
+
+## Screenshots
+
+### Multiple screenshots
+
+Apps should have at least two screenshots. For medium complexity apps 3-6 is a good range, for very large apps with lots of screens 6-10 is appropriate. More than 10 is probably too many.
+
+### Just the app window
+
+Use the "window screenshot" feature in your native system screenshot tool to get just the app window and its shadow. Do not include the wallpaper behind the app, or even the entire desktop.
+
+### Take screenshots on Linux
+
+Do not use screenshots taken on other platforms, especially if it's immediately obvious from window controls and the like.
+
+### Not maximized
+
+Don't use screenshot of windows in maximized or fullscreen state, because in these states windows don't have shadows and rounded corners.
+
+### Include window shadow
+
+Screenshots must include the native toolkit shadow. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this.
+
+### Reasonable window size
+
+In order to ensure that text and interface elements are visible scaled-down in the app store UI the window size should be 1000x700 pixels or smaller (2000x1400 for HiDPI).
+
+### Image captions
+
+Every screenshot should have a caption briefly describing it.
```

Comment:
```suggestion
Every screenshot should have a `caption` briefly describing it.
```
Maybe this allows people to easier see the connection to the `<caption>` keyword in appstream?

Reply:
Works for me. Maybe we could also link to the docs for this?

Reply:
Reworded it, please have a look. Unfortunatly, we can't link to it more directly. 

Their docs also say captions shouldn't be longer then 100 chars

Reply:
No, you can't but you could have posted another one :D

```suggestion
Every screenshot should have a [caption](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-screenshots) briefly describing it. Captions should only be one sentence and not end with a full stop.
```